README.md

```diff
@@ -46,33 +46,54 @@ This is done by getting an actual client
 Here is how you can create a basic client & submit contact.
 
 ```php
-		$contact  = new Contact();
-
-		$contact->set_email( $email );
-		if ( $phone_number != '' ) {
-			$contact->set_phone( $phone_number );
-		}
-		$contact->set_first_name( $first_name );
-		$contact->set_last_name( $last_name );
-		$contact->set_birthday( $birthday );
-		$contact->set_postal_code( $postal_code );
-		$contact->set_address( $address );
-		$contact->set_state( $state );
-		$contact->set_country( $country );
-		$contact->set_city( $city );
-		if ( $email_consent ) {
-			$contact->set_email_consent( 'actual_email_consent_for_gdrp' );
-			$contact->set_email_opt_in( 'where user opted to become subscriber' );
-		}
-
-        $client = \Omnisend\SDK\V1\Omnisend::get_client( 'integration name', 'integration version' );
-
-        $response = $client->create_contact( $contact );
+	$contact  = new Contact();
+
+	$contact->set_email( $email );
+	if ( $phone_number != '' ) {
+		$contact->set_phone( $phone_number );
+	}
+	$contact->set_first_name( $first_name );
+	$contact->set_last_name( $last_name );
+	$contact->set_birthday( $birthday );
+	$contact->set_postal_code( $postal_code );
+	$contact->set_address( $address );
+	$contact->set_state( $state );
+	$contact->set_country( $country );
+	$contact->set_city( $city );
+	if ( $email_consent ) {
+		$contact->set_email_consent( 'actual_email_consent_for_gdrp' );
+		$contact->set_email_opt_in( 'where user opted to become subscriber' );
+	}
+	$client = \Omnisend\SDK\V1\Omnisend::get_client( 'integration name', 'integration version' );
+	
+	$response = $client->create_contact( $contact );
 ```
 
+#### Customer events
+
+Here is how you can send customer events.
+
+```php
+	$contact  = new Contact();
+	$contact->set_email( $email );
+
+	$event =  new Event();
+	$event->set_contact( $contact );
+	$event->set_origin( 'appName' );
+	$event->set_event_name( 'something hapened' );
```

Comment:
if you can suggest something more meaningful, please share

I dont want to add here vent that is our system event, because then I would want to pass all properties and then we will have this huge event here and might confuse users